### PR TITLE
Update XML documentations

### PR DESCRIPTION
**Pull request by pouladpld**

### CODE REVIEW

File: src/Telegram.Bot/ITelegramBotClient.cs [hunk 1]
```diff
@@ -578,21 +578,39 @@ public interface ITelegramBotClient
             string fileId,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Use this method to get download a file. Get <paramref name="filePath"/> by calling <see cref="GetFileAsync"/>
```
Comment (tuscen) [inline src/Telegram.Bot/ITelegramBotClient.cs]: Should be `Use this method to download a file.`

File: src/Telegram.Bot/ITelegramBotClient.cs [hunk 2]
```diff
@@ -578,21 +578,39 @@ public interface ITelegramBotClient
...
+        /// </summary>
+        /// <param name="filePath">Path to file on server</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>File stream</returns>
         Task<Stream> DownloadFileAsync(
             string filePath,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Use this method to get download a file. Get <paramref name="filePath"/> by calling <see cref="GetFileAsync"/>
```
Comment (tuscen) [inline src/Telegram.Bot/ITelegramBotClient.cs]: Same

File: src/Telegram.Bot/ITelegramBotClient.cs (hunk 1 above)
Comment (pouladpld) [reply src/Telegram.Bot/ITelegramBotClient.cs]: Fixed

File: src/Telegram.Bot/ITelegramBotClient.cs (hunk 2 above)
Comment (pouladpld) [reply src/Telegram.Bot/ITelegramBotClient.cs]: Fixed

**Pull request merged**